### PR TITLE
Add pt_BR messages file

### LINKED ISSUE #24: Add translations
Since #23 added the capabilities of translating the extension it would be great to have more translations available for those who aren't native English speakers.

The translations need a folder with the language code in the `_locales` folder and a `messages.json` in that sub-folder. The easiest way to get all the messages is to copy the English folder(`_locales/en/`) and go from there.

Some translations might not need all the fields if e.g. the text is understandable in your language and might only need a translation in languages that use non-latin characters and have their own words/phrases for messages that are otherwise not translated. Messages that aren't translated will take their text from the English translation - see German as an example for that. 

A list of possible translations is available at https://developer.chrome.com/webstore/i18n#localeTable

I'll list translations that are done and won't be needing a translation as checked here. Those currently being worked on or free as unchecked. If it's not on the list but in the possible translation list - see link above - just write a comment under this issue, otherwise write in the linked issue so I can assign you:

- [ ] ar - Arabic #39 - possible solutions #40, #42 and #84 
- [ ] am - Amharic #72
- [x] bg - Bulgarian  #45
- [x] bn - Bengali - #73
- [ ] ca - Catalan #46
- [x] cs - Czech #47
- [x] da - Danish #48
- [x] de - German
- [x] el - Greek
- [x] en - English
- [x] es - Spanish
- [ ] et - Estonian #49
- [ ] fa - Persian #74
- [x] fi - Finnish #50
- [ ] fil - Filipino #51
- [x] fr - French
- [ ] he - Hebrew #52
- [x] hi - Hindi #53
- [x] hr - Croatian #54
- [x] hu - Hungarian #55
- [x] id - Indonesian #56
- [x] it - Italian #57
- [x] ja - Japanese #58
- [x] kn - Kannada #75
- [x] ko - Korean #59
- [x] nl - Dutch
- [x] pl - Polish
- [x] pt_BR - Portuguese - Brazil
- [x] ru - Russian
- [x] tr - Turkish
- [x] uk - Ukrainian #77 
- [x] zh_CN - Chinese (China) #60
- [x] zh_TW - Chinese (Taiwan) #61

### LINKED ISSUE #32: Add pt_BR translations
Add Brazilian Portuguese translations

### PR DESCRIPTION
**Pull request by thalesagapito**
fix #32

### CODE REVIEW

**Comment by izzy** [comment]
Sorry for the hassle but I removed the native notifications in the master, can you get rid of those(type_native_notification) in your translation?

File: _locales/pt_BR/messages.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        "description": "How the converted text will be displayed"
+    },
+    "type_alert": {
+        "message": "Usar alert()",
+        "description": "Setting that enables the messages to be displayed in an alert box"
+    },
+    "type_native_notification": {
+        "message": "Usar notificações nativas",
+        "description": "Setting that enables the messages to be displayed in a native notification"
+    },
```
Comment (thalesagapito) [inline _locales/pt_BR/messages.json:25]:
```suggestion
```

**Comment by thalesagapito** [comment]
> Sorry for the hassle but I removed the native notifications in the master, can you get rid of those(type_native_notification) in your translation?

No problem, it's done

**Comment by izzy** [comment]
Looks good, thanks a lot! 💚✨

**Pull request merged**